src/lib.rs

```diff
@@ -220,6 +220,16 @@
 //! default-features = false
 //! features = ["std"]
 //! ```
+//! 
+//! And to use async-std on `no_std` targets that only support `alloc` only
+//! enable the `core` Cargo feature:
+//!
+//! ```toml
+//! [dependencies.async-std]
+//! version = "1.0.0"
```

Comment:
There is no `alloc` in this version. (Not in latest 1.5.0)
I think this statement is misleading because it can be used in 1.0.0.

Reply:
```suggestion
//! version = "1.5.0"
```